content/working_with/console/pages/dashboard-page.md

```diff
@@ -14,30 +14,20 @@ The Dashboard page is the landing page when you log into the {{< param cfy_conso
 
```

Comment:
As it's not possible to comment on binary files, I'm adding a comment here related to `/images/ui/pages/dashboard-page.png` (referenced in L13).

The top of the page has recently changed and there is another button for services (TF) instead of Getting Started button link:
![image](https://user-images.githubusercontent.com/5202105/149116988-f91920cf-7713-4e7e-8199-1c604d28abe6.png)

Reply:
Updated